### DISCUSSION TITLE
Error: Table not found !

### DISCUSSION THREAD

**Discussion by mrkhoiit** [PostgREST/postgrest#3060] [Q&A]
### Environment

* PostgreSQL version: PostgreSQL 15.4 on x86_64-pc-linux-gnu, compiled by gcc (Debian 10.2.1-6) 10.2.1 20210110, 64-bit)
* PostgREST version: i use docker and use image:  postgrest/postgrest ,  now version: v11.2.1
* Operating system: I run PostgREST on google cloud run service 

### Description of issue
When i create table use `pg `module in nodejs. Create table are successed then i run sql query `NOTIFY pgrst;` but then i use PostREST insert data to table but i recived: 404 Not Found ! and empty object are reponse
```
{}
```
if i wait about 1 or 2 second api working normal . Please help. Thanks !

**Comment by steve-chavez** [comment]
> but then i use PostREST insert data to table but i recived: 404 Not Found ! and empty object are reponse

How did you make the request? Please share a `curl` snippet. Also your table definition .
